metadata-db-app/int-test/cmr/metadata_db/int_test/concepts/tool_association_save_test.clj

```diff
@@ -36,3 +37,14 @@
       (is (= [(str "Tool association could not be associated with provider [REG_PROV]. "
                    "Tool associations are system level entities.")]
              errors)))))
+
+(deftest save-tool-assoc-data-test
```

Comment:
Suggest making the "testing say: ensure that saving worked for an association with a data payload

Reply:
make the text change.